packages/main/src/themes/TextArea.css

```diff
@@ -7,72 +7,112 @@
 :host {
 	width: 100%;
 	color: var(--sapField_TextColor);
+	min-height: var(--__ui5_textarea_min_height);
 	font-size: var(--sapFontSize);
 	font-family: "72override", var(--sapFontFamily);
 	font-style: normal;
-	border-color: var(--sapField_BorderColor);
-	border-radius: var(--sapField_BorderCornerRadius);
 	box-sizing: border-box;
-	line-height: 1.4;
-	letter-spacing: normal;
+	line-height: var(--_ui5_textarea_line_height);
+	Letter-spacing: normal;
 	word-spacing: normal;
 	margin: var(--_ui5_textarea_margin);
 }
 
-:host([disabled]) {
-	opacity: var(--_ui5_input_disabled_opacity);
-	cursor: default;
-	pointer-events: none;
-	border-color: var(--sapField_ReadOnly_BorderColor);
-	-webkit-text-fill-color: var(--sapContent_DisabledTextColor);
-	color: var(--sapContent_DisabledTextColor);
-}
-
-:host([disabled]) .ui5-textarea-inner {
+:host(:not([value-state]):not([readonly]):not([focused])) .ui5-textarea-wrapper {
 	background: var(--sapField_BackgroundStyle);
-	background-color: var(--_ui5_textarea_disabled_background_color);
-}
-
-:host([focused]) .ui5-textarea-inner {
-	outline: var(--_ui5_textarea_focus_outline);
-	outline-offset: var(--_ui5_textarea_focus_outline_offset);
+	background-color: var(--_ui5_textarea_background_color);
```

Comment:
I do not know whether we need parameter for background-color: var(--_ui5_textarea_background_color); when its only value defined is var(--sapField_Background);